libs/shared/typescript/charts-ngx/src/lib/boxplot/boxplot.directive.stories.ts

```diff
@@ -0,0 +1,62 @@
+import {
+  BoxplotProps,
+  CategoryPoint,
+  dynamicBoxplotPoints,
+  staticBoxplotPoints,
+  staticBoxplotSummaries,
+} from '@sagebionetworks/shared/charts';
+import { Meta, StoryObj } from '@storybook/angular';
+import { BoxplotDirective } from './boxplot.directive';
+
+const meta: Meta<BoxplotDirective> = {
+  component: BoxplotDirective,
+  title: 'directives/sageBoxplot',
+  argTypes: {
+    pointTooltipFormatter: {
+      control: { type: 'function' },
+    },
+  },
+  render: (args: BoxplotProps) => ({
+    props: args,
+    template: `<div sageBoxplot [points]="points" [summaries]="summaries" [title]="title" [xAxisTitle]="xAxisTitle" [yAxisTitle]="yAxisTitle" [yAxisMin]="yAxisMin" [yAxisMax]="yAxisMax" [xAxisCategoryToTooltipText]="xAxisCategoryToTooltipText" [pointTooltipFormatter]="pointTooltipFormatter"></div>`,
+  }),
+};
+export default meta;
+type Story = StoryObj<BoxplotDirective>;
+
+export const NoData: Story = {
```

Comment:
```suggestion
export const noData: Story = {
```
Variables are usually lowerCamelCase in TypeScript, or sometimes UPPER_SNAKE_CASE (constants). The same comment applies to other variables created in this PR. Is this a Storybook convention by chance?

Reply:
Yes, the [Storybook docs](https://storybook.js.org/docs/writing-stories#defining-stories) recommend using UpperCamelCase for story exports.